src/foam/u2/grid/GridCell.js

```diff
@@ -236,10 +240,12 @@ foam.CLASS
                 if (this.order){
                     d = d.orderBy(this.order);
                 }
+                //var result = this.ContextualizingSink.create(); 
                 d.select().
                 then(function(result){
                     var div = foam.u2.Element.create('div');
-                    console.log('CELL: row:' + this.rowMatchId + ' col:' + this.colMatchId + ', ' + result.a.length);
+                    console.log('CELL: row:' + this.rowMatchId + ' col:' +
```

Comment:
You shouldn't have logging in production code. At least comment out if you think you'll need it again in the future.